usr/src/apple-touchbar-advanced-0.1/appletbdrm.c

```diff
@@ -130,7 +150,43 @@ struct appletbdrm_fb_request_response {
 	u8 unk_14[12];
 	__le64 timestamp;
 } __packed;
+int appletbdrm_display_pipe_init(struct drm_device *dev,
```

Comment:
Space before this. Also probably just move it together with the other two display_pipe functions above